packages/eslint-plugin/src/configs/base.json

```diff
@@ -1,9 +1,5 @@
 {
   "parser": "@typescript-eslint/parser",
-  "parserOptions": {
-    "sourceType": "module"
-  },
-  "plugins": [
-    "@typescript-eslint"
-  ]
+  "parserOptions": { "sourceType": "module" },
```

Comment:
It's a trivial thing, but I'm just curious why the formatting of this file would have changed in this PR - any idea?

Reply:
the original script just did `JSON.stringify(obj, null, 4)`, but prettier disagreed with that, and reformatted it all the time.

Instead of formatting the stringified + indented json, I just passed the unindented json in, which means this is now formatted exactly how prettier wants!